packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -230,6 +242,30 @@ function useBlockEditorSettings( settings, postType, postId ) {
 
 	const forceDisableFocusMode = settings.focusMode === false;
 
+	const getSectionRootBlock = useCallback( () => {
+		if ( ! contextPostType ) {
+			return null;
+		}
+		const CONTENT_TYPES = [ 'post', 'page' ];
```

Comment:
These are not the only content types so I'd prefer if remove this. I'd rather do the following algorithm instead:

 1- If mode = "template-locked": look for post-content block 
 2- If mode is not "template-locked', look for the "main" group block
 3- fallback to top level. (undefined section container I guess)